starburst-presto/README.md

```diff
@@ -25,27 +24,22 @@ Presto installed:
         --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/starburst-presto/presto.sh
     ```
 
-1.  Once the cluster has been created, Presto is configured to run on port
-    `8080` (though you can change this in the script) on the master node in a
-    Cloud Dataproc cluster. To connect to the Presto web interface, you will
-    need to create an SSH tunnel and use a SOCKS 5 Proxy as described in the
-    [dataproc web interfaces](https://cloud.google.com/dataproc/cluster-web-interfaces)
-    documentation. You can also use the
-    [Presto command line interface](https://docs.starburstdata.com/latest/installation/cli.html)
-    using the `presto` command on the master node.
-
-You can find more information about using initialization actions with Dataproc
-in the [Dataproc documentation](https://cloud.google.com/dataproc/init-actions).
+1.  Presto is configured to run on port `8080` on the cluster'a master node (you can change the
+    port assignment in the script). To connect to the Presto web interface,
+    create an SSH tunnel and use a SOCKS5 Proxy&mdash; see
+    [Dataproc cluster web interfaces](https://cloud.google.com/dataproc/cluster-web-interfaces).
+    You can also use the [`presto` command line interface](https://docs.starburstdata.com/latest/installation/cli.html)
+    on the master node.
 
 ## Important notes
 
-*   This script must be updated based on which Presto version you wish to
-    install
-*   You may need to adjust the memory settings in `jvm.config` based on your
-    needs
-*   Presto is set to use HTTP port `8080` by default, but can be changed using
-    `--metadata presto-port=8060`
-*   Only the Hive connector is configured by default
-*   High-Availability configuration is discouraged as coordinator is started
-    only on `m-0` and other master nodes are idle
+*   Update the script to specify a Presto version to install.
+*   You can adjust the memory settings in `jvm.config`.
+*   By default, Presto uses HTTP port `8080`. Update the
+    script to use a different port, for example,
```

Comment:
Change "Update the script to use a different port" to "Use `presto-port` metadata value to configure a different port"

Reply:
Done.